lib/internal/main/mksnapshot.js

```diff
@@ -3,7 +3,10 @@
 const {
   Error,
   SafeSet,
-  SafeArrayIterator
+  SafeArrayIterator,
+  ObjectGetOwnPropertyDescriptor,
+  ObjectDefineProperty,
+  ObjectSetPrototypeOf
```

Comment:
nit: ASCII order and trailing comma.
```suggestion
  ObjectDefineProperty,
  ObjectGetOwnPropertyDescriptor,
  ObjectSetPrototypeOf,
  SafeArrayIterator,
  SafeSet,
```